app/services/financial_year_close.rb

```diff
@@ -11,16 +11,60 @@ def initialize(year, to_close_on, options = {})
     @options = options
   end
 
+  def benchmark(message)
```

Comment:
Was the close properly tested with a big amount of data to confirm performance?

Reply:
I don't know, @ionosphere was in charge of testing the closure, should we wait for him to tell us if we remove it or not ?